content/docs/command-reference/exp/run.md

```diff
@@ -0,0 +1,108 @@
+
+## Synopsis
+
+```usage
+usage: dvc exp run [-h] [-q | -v] [-f]
+                   [<repro_options> ...]
+                   [--params [<filename>:]<params_list>]
+                   [-n <name>] [--queue] [--run-all] [-j <number>]
+                   [targets [targets ...]]
+
+positional arguments:
+  targets               Stages to reproduce. 'dvc.yaml' by default.
+```
+
+## Description
+
+Provides a way to execute and track `dvc experiments` in your
+<abbr>project</abbr> without polluting it with unnecessary commits, branches,
+directories, etc.
+
+> `dvc exp run` is equivalent to `dvc repro` for <abbr>experiments</abbr>. It
+> has the same behavior when it comes to stage execution (restores the
+> dependency graph, etc.). It differs from in that experiment dependencies and
+> outputs will always be <abbr>cached</abbr> (so they can be restored later).
+
+Each `dvc exp run` creates a variation based on the last project version (Git
+commit) and stores it internally with an automatic experiment name like
+`exp-bfe64` (which can be customized with the `--name` option). The results of
+the last experiment can be seen in the <abbr>workspace</abbr>. To display and
+compare your experiments, use `dvc exp show` or `dvc exp diff`.
+
+<details>
+
+### How does DVC track experiments?
+
+`dvc exp` uses actual commits under custom
+[Git references](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+(found in `.git/refs/exps`). Each commit has the Git `HEAD` as parent and has
+it's own SHA-256 hash. These are not pushed to the Git remote by default (see
+`dvc exp push`).
+
+> References have a unique signature similar to the
+> [entries in the run-cache](/doc/user-guide/project-structure/internal-files#run-cache).
+
+</details>
+
+Successful experiments can be made
+[persistent](/doc/user-guide/experiment-management#persistent-experiments) by
+committing them to the Git repo. You can use `dvc exp apply` to roll back the
+workspace to a previous experiment first. Unnecessary ones can be removed with
+`dvc exp gc`, or abandoned (and their data with `dvc gc`).
+
+## Parallel execution
+
+⚠️ Multi-threading is experimental and may be unstable. ⚠️
+
+When using `-j` (`--jobs`) to run experiments in parallel, make sure you're
+using a number of jobs that your machine or environment can handle.
+
+Note that DVC makes no attempt to intelligently schedule or prioritize stages
+within pipelines.
+
+## Options
+
+> In addition to the following options, `dvc exp run` also accepts the same
+> options as `dvc repro`, with the exception that `--no-commit` has no effect
+> here.
+
+- `--params [<filename>:]<params_list>]` - reproduce an experiment using the
+  specified [parameter](/doc/command-reference/params) values. Accepts a
+  comma-separated list of key-value pairs in the form
+  `[filename:]key1=val1,key2=val2...`. Any specified values will override those
+  from the contents of the relevant params file.
```

Comment:
Feel free to edit the description, but this option name and format changed.

```suggestion
`-S [<filename>:]<param_name>=<param_value>, --set-param [<filename>:]<param_name>=<param_value>` - 
Use the specified param value when reproducing pipelines.
```

Reply:
Oh thanks ya I still need to double check all the options 😱

Reply:
Forgot to let you know that all the options have been reviewed now @dberenbaum